go.mod

```diff
@@ -206,6 +206,9 @@ replace google.golang.org/protobuf => github.com/lightninglabs/protobuf-go-hex-d
 // Temporary replace rule until sqldb is tagged.
 replace github.com/lightningnetwork/lnd/sqldb => ./sqldb
 
+// Temporary replace rule until kvdb is tagged.
+replace github.com/lightningnetwork/lnd/kvdvb => ./kvdb
```

Comment:
This seems to be misspelled... Weird that nothing seems to break because of it? Or maybe the replace doesn't take effect after all and we're still running with the old code?